src/inmanta/app.py

```diff
@@ -84,6 +84,15 @@ def server_parser_config(parser: argparse.ArgumentParser, parent_parsers: abc.Se
         help="Maximum time in seconds the server will wait for the database to be up before starting. "
         "A value of 0 means the server will not wait. If set to a negative value, the server will wait indefinitely.",
     )
+    parser.add_argument(
+        "--compatibility-file",
+        type=str,
+        dest="compatibility_file",
+        help="Path to the compatibility.json file. During startup, the server will perform version compatibility check "
+        "for the PostgreSQL version being used. For more information about this file, please refer to "
+        "https://docs.inmanta.com/inmanta-service-orchestrator/latest/reference/compatibility.html.",
```

Comment:
This links to the documentation of the latest iso version. It's not necessarily the one for the running orchestrator. I am also not sure whether we want to make the server log a url to our docs. If we ever change the path for some reason, we will forget to update the link in the code. It's probably better to mention to consult the documentation without offering a concrete url.